nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -238,6 +238,13 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - `unrar` was updated to v7. See [changelog](https://www.rarlab.com/unrar7notes.htm) for more information.
 
+- `zx` was updated to v8, which introduces several breaking changes, namely:
+  `$.verbose` now defaults to `false` (but errors are still printed to `stderr`);
+  `ssh` API was dropped in favor of using the [webpod](https://github.com/webpod/webpod) package;
+  `powershell` is not automatically setup on Windows by default anymore;
```

Comment:
is this relevant, since nixpkgs doesn't support windows ?

Reply:
@kirillrdy not gonna lie, I actually laughed out loud when I read your comment; I rewrote the changelog and didn't even give it a second thought. You're obviously right.

On a separate note: should I just link the changelog and avoid summarizing the changes themselves entirely? I saw a mixed of both approaches in the release notes.